test/Calibur.execute.t.sol

```diff
@@ -53,6 +53,12 @@ contract CaliburExecuteTest is TokenHandler, HookHandler, ExecuteFixtures, Deleg
         nonce = key << 64 | seq;
     }
 
+    function test_supportsExecutionMode() public {
+        // Test ERC-7821 supportsExecutionMode
+        assertEq(signerAccount.supportsExecutionMode(BATCHED_CALL), true);
+        assertEq(signerAccount.supportsExecutionMode(BATCHED_CAN_REVERT_CALL), true);
```

Comment:
should we add the failure modes? or fuzz this and assert false for anything except these two modes

Reply:
Yeah I'll fuzz this